docs/how-to-guides/integrating-autoware/tuning-parameters-and-performance/important-temporal-performance-metrics.md

```diff
@@ -115,7 +115,7 @@ The following list shows the temporal performance metrics around Obstacle segmen
 
 | ID       | Representation in the model              | Metric meaning                                                                           | Related functionality | Reference value | Reason to choose it as a metric                                                    | Note |
 | -------- | ---------------------------------------- | ---------------------------------------------------------------------------------------- | --------------------- | --------------- | ---------------------------------------------------------------------------------- | ---- |
-| OSEG-001 | **Message rate** from CPS #4 to CPS #7   | Update rate of occupancy grid map received by Planning (`behavior_path_planner`)         | Obstacle segmentation | 10 Hz           | Occupancy grid map must be updated frequently and smoothly.                        |      |
-| OSEG-002 | **Response time** from CPS #0 to CPS #9  | Processing time to output occupancy grid map after LiDARs output sensing data.           | Obstacle segmentation | N/A             | Planning relies on fresh and up-to-date perceived data from occupancy grid map.    |      |
-| OSEG-003 | **Message rate** from CPS #6 to CPS #11  | Update rate of obstacle segmentation received by Planning (`behavior_velocity_planner`). | Obstacle segmentation | 10 Hz           | Obstacle segmentation must be updated frequently and smoothly.                     |      |
-| OSEG-004 | **Response time** from CPS #0 to CPS #11 | Processing time to output obstacle segmentation after LiDARs output sensing data.        | Obstacle segmentation | N/A             | Planning relies on fresh and up-to-date perceived data from Obstacle segmentation. |      |
+| OSEG-001 | **Message rate** from CPS #4 to CPS #7   | Update rate of Occupancy grid map received by Planning (`behavior_path_planner`)         | Obstacle segmentation | 10 Hz           | Planning relies on occupancy grid map to be updated frequently and smoothly for creating accurate trajectory.                        |      |
+| OSEG-002 | **Response time** from CPS #0 to CPS #9 via CPS #7 | Response time to consume Occupancy grid map after LiDARs output sensing data.           | Obstacle segmentation | N/A             | Planning relies on fresh and up-to-date perceived data from occupancy grid map for creating accurate trajectory..    |      |
+| OSEG-003 | **Message rate** from CPS #6 to CPS #11  | Update rate of obstacle segmentation received by Planning (`behavior_velocity_planner`). | Obstacle segmentation | 10 Hz           | Obstacle segmentation must be updated frequently and smoothly for creating accurate trajectory..                     |      |
```

Comment:
It would be better to use the same sentence format as OSEG-001 like: "Planning relies on ..."